src/Dfc.CourseDirectory.Database/Pttcd/Views/vwProviderSearchIndex.sql

```diff
@@ -1,19 +1,19 @@
-create
-    --or alter -- not supported by dacpac
-    view Pttcd.vwProviderSearchIndex
-as
-select
-    p.ProviderId,
-    p.Ukprn,
-    p.ProviderName,
-    p.ProviderStatus,
-    p.UkrlpProviderStatusDescription,
-    p_contact.AddressPostcode Postcode,
-    p_contact.AddressItems Town, -- for approximate compatibility, the old index was reading Items[0] which has been squashed into this field in sql
-    p.Version -- Timestamp (RowVersion) type, used here for cosmos HighWaterMark
-from Pttcd.Providers p
-outer apply (
-    select top 1 pc.ProviderId, pc.ContactType, pc.AddressItems, pc.AddressPostTown, pc.AddressPostcode
-    from Pttcd.ProviderContacts pc
-    where pc.ProviderId = p.ProviderId and pc.ContactType = 'P'
-) p_contact
+CREATE VIEW [Pttcd].[vwProviderSearchIndex]
+AS
+SELECT      p.ProviderId,
+            p.Ukprn,
+            p.ProviderName,
+            p.ProviderStatus,
+            p.UkrlpProviderStatusDescription,
+            pc.AddressPostcode Postcode,
+            ISNULL(pc.AddressPostTown, pc.AddressItems) Town,
```

Comment:
If we've now fixed the source of the missing data in #1942 do we still need the `isnull` fallback to `AddressItems` or can we always use `AddressPostTown`. I *think* it'll be fine without the fallback.